File: FE/FE-EVRental/replace-colors.js
import { readFileSync, writeFileSync } from 'fs';
import { join, dirname } from 'path';
import { fileURLToPath } from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = dirname(__filename);

const filePath = join(__dirname, 'src', 'styles', 'Staff.css');

// Đọc file
let content = readFileSync(filePath, 'utf8');

// Thay thế màu tím/xanh dương thành xanh lá cây nhạt
content = content.replace(/#667eea/g, '#66c9a9');
content = content.replace(/#764ba2/g, '#56ab91');
content = content.replace(/rgba\(102, 126, 234/g, 'rgba(102, 201, 169');

// Ghi lại file
writeFileSync(filePath, content, 'utf8');

console.log('✅ Đã thay đổi màu sắc thành công!');
console.log('   Tím (#667eea) → Xanh lá (#66c9a9)');
console.log('   Tím đậm (#764ba2) → Xanh lá đậm (#56ab91)');

File: replace_colors.py
import re

# Đọc file
file_path = r'FE\FE-EVRental\src\styles\Staff.css'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Thay thế màu
content = content.replace('#667eea', '#66c9a9')
content = content.replace('#764ba2', '#56ab91')
content = content.replace('#1e3c72', '#56ab91')
content = content.replace('#2a5298', '#66c9a9')
content = content.replace('rgba(102, 126, 234', 'rgba(102, 201, 169')

# Ghi lại file
with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print("✅ Đã thay đổi màu thành công!")
print("Tím/Xanh dương → Xanh lá cây nhạt")
